server/src/main/java/org/apache/iotdb/db/engine/tsfilemanagement/level/LevelTsFileManagement.java

```diff
@@ -209,25 +232,25 @@ public void removeAll(List<TsFileResource> tsFileResourceList, boolean sequence)
   public void add(TsFileResource tsFileResource, boolean sequence) {
     long timePartitionId = tsFileResource.getTimePartition();
     int level = getMergeLevel(tsFileResource.getTsFile());
-    if (level <= maxLevelNum - 1) {
-      if (sequence) {
+    if (sequence) {
+      if (level <= maxLevelNum - 1) {
         sequenceTsFileResources
             .computeIfAbsent(timePartitionId, this::newSequenceTsFileResources).get(level)
             .add(tsFileResource);
       } else {
-        unSequenceTsFileResources
-            .computeIfAbsent(timePartitionId, this::newUnSequenceTsFileResources).get(level)
+        sequenceTsFileResources
+            .computeIfAbsent(timePartitionId, this::newSequenceTsFileResources).get(maxLevelNum - 1)
             .add(tsFileResource);
```

Comment:
any difference between this if , else，I think both can use `get(level)`

Reply:
add comment